common/libs/wallet/deterministic/trezor.ts

```diff
@@ -53,28 +80,21 @@ export class TrezorWallet extends DeterministicWallet implements IFullWallet {
 
   public signMessage = () => Promise.reject(new Error('Signing via Trezor not yet supported.'));
 
-  public displayAddress = (dPath?: string, index?: number): Promise<any> => {
-    if (!dPath) {
-      dPath = this.dPath;
-    }
-    if (!index) {
-      index = this.index;
-    }
-
-    return new Promise((resolve, reject) => {
-      (TrezorConnect as any).ethereumGetAddress(
-        dPath + '/' + index,
+  public displayAddress(): Promise<any> {
+    return new Promise(resolve => {
+      TrezorConnect.ethereumGetAddress(
+        this.dPath + '/' + this.index,
```

Comment:
Mostly stylistic, but string template here would be better

Reply:
👍 will clean these up in #1946.